packages/dds/tree/src/test/rebase/findAncestor.spec.ts

```diff
@@ -0,0 +1,168 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+
+// Allow importing from these specific files which are being tested:
+/* eslint-disable-next-line import/no-internal-modules */
+import { findAncestor, findCommonAncestor } from "../../core/rebase";
+
+interface Node {
+	parent?: Node;
+}
+
+function node(parent?: Node): Node {
+	return {
+		parent,
+	};
+}
+
+function assertAncestor(
+	descendant: Node,
+	expectedAncestor: Node | undefined,
+	expectedPath?: Node[],
+): void {
+	const path: Node[] = [];
+	const foundAncestor = findAncestor([descendant, path], (n) => n === expectedAncestor);
+	assert.strictEqual(foundAncestor, expectedAncestor);
```

Comment:
The fact that you imported `assert` the way you did means you can drop the `strict` portion of the method name.

Reply:
oh, nice, did not know that. Thanks!